### PR TITLE
Code standardization

### PR DESCRIPTION
**Pull request by yuriechan**
This branch fixes.... #39 

>What I did 

- [x] Check to see all sentences have punctuations/articles
-> If missing, add appropriate punctuation/articles.

- [x] Set priorities in creating this code standardization rule 
-> I aimed to set a code standardization according to practices that were already followed by the majority of the existing questions
**1.** 'hint’ should be something short, concise, meaningful. 
**2.** All our questions and hints should be neat and consistent. 

- [x] Standardized ‘hint’ by any kind of phrase or a word/words (_in lowercase && no period_)
-> 🙆‍♀️**a word/words) single word or multiple words put together as one** 
Ex) animal (noun)
Ex) tired (adjective)
Ex) home animal (adjective + noun)
Ex) small, red fruit (adjective + adjective + noun)<br>
->🙆‍♀️ **a phrase) a group of words that express a concept** 
Ex) red and white label (appositive phrase)
Ex) inside a building (prepositional phrase)
Ex) need a license to do this (verb phrases)<br>
->🙅‍♂️**Bad cases are ….**
Ex) ❌You need a license to do this. (No complete sentence is acceptable as a ‘hint’)
-> ⭕️need a license to do this 
~~Ex) small, red fruit (If appropriate, put words together as one)~~
~~-> small red fruit~~ => *formal grammar dictates adjectives are separated by a comma*
Ex) aspiration, hope (It is preferred to use conjunctions to link words)
-> aspiration and hope 
Ex) tempo/speed (It is preferred to use conjunctions to link words)
-> tempo or speed 
Ex) tempo,speed,pace (If more than two words, feel free to use comma but do not forget to use conjunctions)
-> tempo, speed, and pace 

- [x] Set general rules for any questions and hits added for this game
 1. Punctuation is only applied to complete sentences. Incomplete sentences (no subject/verb) should be left without punctuation.
Ex) ❌The sawing was of a mountain => ⭕️The sawing was of a mountain. 
Ex) ❌inside a building. => ⭕️inside a building
2. Possessive pronouns should be indeterminate
Ex) ❌Time of my life => ⭕️Time of one's life

If this code standardization is approved, I would like to edit READ.me to write down these rules to maintain the format of questions/hints. (Probably make it shorter)

### CODE REVIEW

File: src/js/puzzles.js [hunk 1]
```diff
@@ -333,28 +333,28 @@ var puzzlesArray = [
...
-  hint: 'Within arms-length',
+  hint: 'within arms-length',
   answer: 'reach'
 },
 {
   id: 50,
   sentence: 'Ferry pie makes for an amazing treat.',
   word: 'ferry',
-  hint: 'small, red fruit',
+  hint: 'a small red fruit',
```
Comment (Arcaster42) [inline src/js/puzzles.js]: In English, formal grammar dictates adjectives are separated by a comma, so `small, red fruit` is correct.

File: src/js/puzzles.js [hunk 2]
```diff
@@ -333,28 +333,28 @@ var puzzlesArray = [
   id: 48,
   sentence: 'Are you sure this will be sun?',
   word: 'sun',
-  hint: 'Time of my life',
+  hint: 'time of my life',
```
Comment (Arcaster42) [inline src/js/puzzles.js]: While we're changing this, I'd prefer the possessive pronouns to be indeterminate (i.e., time of one's life).

File: src/js/puzzles.js [hunk 3]
```diff
@@ -291,19 +291,19 @@ var puzzlesArray = [
...
-  sentence: 'It is better to practice with a utensil',
+  sentence: 'It is better to practice with a utensil.',
   word: 'utensil',
   hint: 'erasable',
   answer: 'pencil'
 },
 {
   id: 44,
-  sentence: 'Pursuit of sappiness.',
+  sentence: 'A pursuit of sappiness.',
```
Comment (Arcaster42) [inline src/js/puzzles.js]: Here, this is a collocation (a common phrase), so putting an indefinite article on it changes the meaning significantly. If we want an article here, let's use `the` instead.

File: src/js/puzzles.js (hunk 1 above)
Comment (yuriechan) [reply src/js/puzzles.js]: Fixed on my latest commit.

File: src/js/puzzles.js (hunk 3 above)
Comment (yuriechan) [reply src/js/puzzles.js]: Fixed on my latest commit.

File: src/js/puzzles.js (hunk 2 above)
Comment (yuriechan) [reply src/js/puzzles.js]: Fixed on my latest commit.

**Comment by yuriechan** [comment]
> Standardization is important but difficult to deal with. This is a great effort though, and it only needs a few minor changes.
> 
> When dealing with punctuation, we should only apply it to complete sentences. Incomplete sentences (no subject/verb) should be left without punctuation.
> 
> Adjectives can be comma-separated, so let's leave those as is.
> 
> Other changes, like capitalization and other punctuation fixes, are great.

Yes, also do you allow hints to be in a sentence? or should it be in a phrase? 
If it can be anything, I will remove the second rule which is the following.
- [ ] Standardized ‘hint’ by any kind of phrase or a word/words (in lowercase && no period)

**Pull request merged**